docs/content/guides/traffic_management/listener_configuration/http_connection_manager/dfp/_index.md

```diff
@@ -57,6 +57,68 @@ spec:
             autoHostRewriteHeader: "x-rewrite-me" # host header will be rewritten to the value of this header
 ```
 
+## Set circuit breakers for dynamically discovered upstreams {#dfp-circuit-breakers}
+
+By default, Envoy allows a maximum of 1024 connections to dynamically discovered upstreams. Starting in Gloo Gateway Enterprise version 1.19.3, you can override these settings by defining custom circuit breakers for your dynamic forward proxy. 
+
+1. Update your gateway proxy to add custom circuit breaker settings for your dynamic forward proxy. The following example overrides the default number of connections, requests, pending requests, and retries to 40,000. For more information, see the [Circuit breaker configuration]({{% versioned_link_path fromRoot="/reference/api/github.com/solo-io/gloo/projects/gloo/api/v1/circuit_breaker/circuit_breaker.proto.sk/#circuitbreakerconfig" %}})
+
+   ```bash
+   kubectl -n gloo-system patch gw/gateway-proxy --type merge -p "
+   spec:
+     httpGateway:
+       options:
+         dynamicForwardProxy: 
+           circuitBreakers:
+             maxConnections: 40000
+             maxPendingRequests: 40000
+             maxRequests: 40000
+             maxRetries: 40000
+             trackRemaining: true
+   "
+   ```
+
+   | Setting | Description | 
+   | -- | -- | 
+   | `maxConnections` | The maximum number of connections that Envoy makes to the upstream cluster. If not specified, the default is 1024. | 
+   | `maxPendingRequests` | The maximum number of pending requests that Envoy allows to the upstream cluster. If not specified, the default is 1024.| 
+   | `maxRequests` | The maximum number of parallel requests that Envoy makes to the upstream cluster. If not specified, the default is 1024.| 
+   | `maxRetries` | The maximum number of parallel retries that Envoy allows to the upstream cluster. If not specified, the default is 3. | 
```

Comment:
Just curious, if the default is only 3, is 40,000 the same as the other values realistic? Or is the default the same 1024 as the others?

Reply:
no yeah it is 3 :D 

but the field was testing it with 10 million and it worked. lol